src/main/java/io/cdap/plugin/gcp/bigquery/sink/BigQuerySinkUtils.java

```diff
@@ -585,13 +585,19 @@ public static String generateUpdateUpsertQuery(Operation operation,
                                                        formatPartitionFilter(partitionFilter), criteria) : criteria;
     String fieldsForUpdate = tableFieldsList.stream().filter(s -> !tableKeyList.contains(s))
       .map(s -> String.format(CRITERIA_TEMPLATE, s, s)).collect(Collectors.joining(", "));
+
+    orderedByList.replaceAll(s -> "`" + Arrays.stream(s.split(" ")).map(String::trim)
```

Comment:
What is this code doing? If you just need to quote characters, you can do this by using;

```orderedByList = orderedByList.stream().map(s -> "`" + s + "`").collect(Collectors.toList());```

Reply:
Done